lib/utils.ts

```diff
@@ -141,23 +191,23 @@ function getRuleExportsESM(ast, scopeManager) {
           const nodes =
             statement.declaration.type === 'VariableDeclaration'
               ? statement.declaration.declarations.map(
-                  (declarator) => declarator.init,
+                  (declarator) => declarator.init!,
```

Comment:
Instead of forcing this to exist, can we filter out null/undefined afterward with `.filter(val=>!!val)`?

Reply:
Updated